airbyte-integrations/bases/standard-destination-test/src/main/resources/exchange_rate_messages.txt

```diff
@@ -1,8 +1,8 @@
 {"type": "RECORD", "record": {"stream": "exchange_rate", "emitted_at": 1602637589000, "data": { "date": "2020-08-29T00:00:00Z", "NZD": 0.12, "HKD": 2.13}}}
 {"type": "STATE", "state": { "data": {"start_date": "2020-08-31"}}}
-{"type": "RECORD", "record": {"stream": "exchange_rate", "emitted_at": 1602637589100, "data": { "date": "2020-08-30T00:00:00Z", "NZD": 1.14, "HKD": 7.15}}}
```

Comment:
why is this file changing?

Reply:
I am modifying this file to make the tests more stable across destinations by making sure that we have different `emitted_at` dates so that the tests can do `select * from XXX order by emitted_at` and retrieve the list of records in the exact same order as the text file (which makes comparisons tests easier to handle)